test/grammars/external-lexer/grammar.ne

```diff
@@ -0,0 +1,5 @@
+# Use %token to match any token of that type instead of "token":
+multiplication -> %number %ws %times %ws %number {% ([first, , , , second]) => first * second %}
```

Comment:
lexer tokens need a `%` in front of them. This is not super intuitive, so I'll add some documentation somewhere (either existing grammar files or hitchhikers guide).